lib/ansible/modules/cloud/misc/proxmox_kvm.py

```diff
@@ -19,7 +19,7 @@
 
 ANSIBLE_METADATA = {'status': ['preview'],
                     'supported_by': 'community',
-                    'version': '1.0'}
```

Comment:
This is the version of the metadata, not your file. So no need to change that. 

It's on my to do list to document ANSIBLE_METADATA so I'll add a note about this.